src/DssSpell.sol

```diff
@@ -19,22 +19,29 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3MAavePoolConfig, D3MAaveBufferPlanConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
-
-interface PoolConfiguratorLike {
-    function setReserveFreeze(address asset, bool freeze) external;
-    function setReserveInterestRateStrategyAddress(address asset, address newRateStrategyAddress) external;
-    function setReserveFactor(address asset, uint256 newReserveFactor) external;
+interface StarknetLike {
+    function setCeiling(uint256 _ceiling) external;
+}
+
+interface GemLike {
+    function transfer(address, uint256) external returns (bool);
+    function allowance(address, address) external view returns (uint256);
+    function approve(address, uint256) external returns (bool);
+}
+
+interface VestLike {
+    function file(bytes32, uint256) external;
+    function restrict(uint256) external;
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function yank(uint256) external;
```

Comment:
Please use `cast interface` for 0x6D635c8d08a1eA2F1687a5E46b666949c977B7dd (VEST) and 0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2 (MKR) and ensure that the function specs match the results you get.